os/arch/arm/Kconfig

```diff
@@ -79,6 +79,7 @@ config ARCH_CHIP_STM32H745
           select ARCH_HAVE_MPU
           select ARCH_HAVE_I2CRESET
           select ARCH_HAVE_HEAPCHECK if DEBUG
+		  select BOARDCTL_RESET_HEADER
```

Comment:
minor change, could you please align the indentation to match the above line.

Reply:
Fixed! Thanks.